tests/basis_test.py

```diff
@@ -67,6 +67,14 @@ def test_cheb_call_random_points():
         f = ChebyshevFirstKind(n)
         assert numpy.isclose(f(x),special.eval_chebyt(n,x))
 
+def test_cheb_call_random_points_multi_input():
```

Comment:
You don't need to run this test on multiple `ns`. Just choose one, unless there are special cases and then include those explicitly.